app/src/main/java/com/pluu/support/ktoon/OllehDetailApi.kt

```diff
@@ -4,11 +4,8 @@ import android.net.Uri
 import com.pluu.kotlin.asSequence
 import com.pluu.support.impl.AbstractDetailApi
 import com.pluu.support.impl.REQUEST_METHOD
-import com.pluu.webtoon.di.NetworkModule
-import com.pluu.webtoon.item.Detail
-import com.pluu.webtoon.item.DetailView
-import com.pluu.webtoon.item.Episode
-import com.pluu.webtoon.item.ShareItem
+import com.pluu.webtoon.di.NetworkUseCase
+import com.pluu.webtoon.item.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Pluu/WebToon/blob/45d44b0b109e4d385144e8b26bdd4efb5d452d37/app/src/main/java/com/pluu/support/ktoon/OllehDetailApi.kt#L8"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>